CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # PD Change Log
+
+## v2.0.0-GA
+### New Features
+* Support using pd-ctl to scatter spcified Regions for manual adjust hotspot regions in some cases
+### Improves
+* More configuration check rules to prevent unreasonable configurations
```

Comment:
"configurations" -> "configuration"

The "configuration" is an uncountable noun here when it is used for this meaning.